main/guides/js-programming/README.md

```diff
@@ -0,0 +1,38 @@
+# Agoric JavaScript Programming Extensions
+
+Agoric's platform lets you write secure smart contracts in JavaScript. The platform itself is mainly written in JavaScript. However, we've made several Agoric-specific additions to general JavaScript programming that you should know about and understand before programming on the platform. Some are *concepts*, others are *Agoric library additions*, and some are at the *syntax level*. All changes at the language level are in process to become official standards.
+
+Extensions covered in this document are:
+- **[`BigInt`](./bigint.md)**: JavaScript's `Number` primitive only represents
```

Comment:
We should probably make it clear that we didn't create BigInts - they are already part of JavaScript, just a newer part.